include/onnxruntime/core/session/onnxruntime_cxx_inline.h

```diff
@@ -1201,6 +1201,45 @@ inline void* CustomOpApi::KernelContext_GetGPUComputeStream(const OrtKernelConte
   return out;
 }
 
+inline void CustomOpApi::CreateAttribute(_In_ const char* name,
+                                         _In_ const void* data,
+                                         _In_ int len,
+                                         _In_ ONNXTensorElementDataType type,
```

Comment:
>ONNXTensorElementDataType [](http://example.com/codeflow?start=46&length=25)

Question for every reviewer: do u think it is better we create an Enum type in C API regarding attr types?
For now we just reuse the tensor type which lacks the ability to differentiate int and ints, float and floats.

Reply:
Created enum OrtOpAttrType.